iceoryx_hoofs/platform/win/source/socket.cpp

```diff
@@ -15,45 +15,6 @@
 // SPDX-License-Identifier: Apache-2.0
```

Comment:
Can this be deleted? It has only stub functions anyway.

Reply:
It could but why?

Reply:
to prevent people from using non-functional unix domain sockets on windows?

Reply:
I added warning on `stderr` when the socket functions are used so it is visible to the user.